content/education/manage-coursework-with-github-classroom/get-started-with-github-classroom/about-github-classroom.md

```diff
@@ -0,0 +1,41 @@
+
+## About {% data variables.product.prodname_classroom %}
+
+{% data variables.product.prodname_classroom %} is a teaching tool that lets teachers and school administrators create and manage digital classrooms and assignments. You can create assignments for individual students or groups, set due dates, and track assignments on your teacher dashboard. Additionally, {% data variables.product.prodname_classroom %} has many features that simplify tasks like providing feedback, grading assignments, and integrating your existing teaching tools.
+
+## {% data variables.product.prodname_classroom %} features 
+
+{% data variables.product.prodname_classroom %} offers a variety of features to simplify teaching and learning.
+
+### Assignment templates
+
+With {% data variables.product.prodname_classroom %}, you can create assignments that use template repositories with boilerplate code, documentation, and other resources. Assignments with template repositories create assignment repositories with starter code for your students. For more information, see "[AUTOTITLE](/education/manage-coursework-with-github-classroom/teach-with-github-classroom/create-an-assignment-from-a-template-repository)."
+
+### Autograding
+
+Through {% data variables.product.prodname_classroom %}, you can configure tests to automatically grade students' work each time they push to their assignment repository. To learn more about autograding with {% data variables.product.prodname_classroom %}, see "[AUTOTITLE](/education/manage-coursework-with-github-classroom/teach-with-github-classroom/use-autograding)."
+
+### Connect a learning management system
+
+Optionally, you can connect a learning management system (LMS) to {% data variables.product.prodname_classroom %} to import a student identifier roster for your classroom. For more information, see "[AUTOTITLE](/education/manage-coursework-with-github-classroom/teach-with-github-classroom/connect-a-learning-management-system-course-to-a-classroom)."
+
+### Feedback pull requests
+
+With {% data variables.product.prodname_classroom %}, you can enable feedback pull requests. If you enable this feature for an assignment, {% data variables.product.prodname_classroom %} creates a special pull request titled **Feedback** in the assignment repository for each student or group. For more information, see "[AUTOTITLE](/education/manage-coursework-with-github-classroom/teach-with-github-classroom/leave-feedback-with-pull-requests)."
+
+### Integrate a classroom with an integrated development environment (IDE)
+
+In {% data variables.product.prodname_classroom %}, you can integrate a classroom with an IDE. Students can make changes to their assignment repositories through the IDE, or by checking out and running code locally with the necessary software. For more information, see "[AUTOTITLE](/education/manage-coursework-with-github-classroom/integrate-github-classroom-with-an-ide/integrate-github-classroom-with-an-ide)."
+
+## Further reading
```

Comment:
Usually, but this might have changed, we add articles that are not in the same map topic as the current article to this section, so I would suggest removing this link, and linking to articles that customers may be interested in, having read this overview of GitHub Classroom. I would look at @saritai's suggestions for inspiration 😃 

> this section should direct users where to go to get started and learn more detailed information - we should include links to the major sections/TOCs. So we could include a link to "[GitHub Classroom basics](https://docs.github.com/en/education/manage-coursework-with-github-classroom/basics-of-setting-up-github-classroom)" page for if they want to view videos on how to get started. https://docs.github.com/en/education/manage-coursework-with-github-classroom/basics-of-setting-up-github-classroom. We should also include a link to “[Teach with GitHub Classroom](https://docs.github.com/en/education/manage-coursework-with-github-classroom/teach-with-github-classroom)” for teachers to get started setting up classrooms and assignment, and also a link to “[Integrate GitHub Classroom with an IDE](https://docs.github.com/en/education/manage-coursework-with-github-classroom/integrate-github-classroom-with-an-ide)” if they want to learn more about that. And for students who want to learn more about the GitHub Classroom learning experience, we can link to “[Learn with GitHub Classroom](https://docs.github.com/en/education/manage-coursework-with-github-classroom/learn-with-github-classroom)”.